### ISSUE TITLE
Update the admin page

### ISSUE THREAD

**Issue by KristianMSchmidt** [DatalogiForAlle/Markedsspillet#111]
I think I'm the only one who ever used the admin area. And even I haven't used it for a long time - partly because it requires creating a new superuser with each new container during development. However, I think it would be nice to have a good admin page when the application is live. So we should update the admin page at some point to make it reflect the current state of the project. It shouldn't take long.

**Comment by dybber** [comment]
I think I will make this a "nice to have". Let's focus on other things for now.

**Comment by dybber** [comment]
But feel free to think about how this could be done, what it should include, how it should look etc. and add whatever you come up with to the issue, so we can discuss it a bit more before starting.

**Comment by KristianMSchmidt** [comment]
Ideally, perhaps, but in case there is a database related error, I think it can be nice to have the admin's quick overview of the generated data. Also, for a curious developer, who doesn't have access to the server, the admin is a nice way to check if and how the application is being used. 

Updating the admin is really just about adding the new database fields to the list of fields being shown on the admin page -- it's an update job, that shouldn't take more than 15 minutes.  Perhaps I will reopen at some point.

**Comment by KristianMSchmidt** [comment]
I will reopen in order not to forget. For the admin to serve any purpose we also need to create some super user accounts on the production database. Let's discuss it next week...
